### PR TITLE
Hello Microverse Basic-setup-Repo

### PR DESCRIPTION
**Pull request by cheronojudith**
Added Html file, CSS file, and setup linters

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (GedeonTS) [inline .gitignore:1]: - [ ] As requested by the previous reviewer [here](https://github.com/cheronojudith/module1-first-project/pull/3#:~:text=Nice%20work%20creating,and%20test.md), Kindly make sure test.md is also ignored as node_module :recycle: 

__Kindly make sure to use the same PR for review, as it makes it easier to track the project's evolution :recycle:__

**Review by GedeonTS — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED :recycle: :
Hi @cheronojudith , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights :tophat: :+1: 
- No linter errors :+1: 
- Good use of Github flow :+1: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions :ringed_planet: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore (hunk 1 above)
Comment (cheronojudith) [reply .gitignore:1]: Thank you @GedeonTS for correcting me have ignored the test.md

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+test.md
```
Comment (adel-gu) [inline .gitignore:2]: Well done adding the "test.md" to the ".gitignore" 👏🏼 however the `test.md` is still appearing in the repo root directory as shown in the following screenshot
image

- [ ] Please ensure that the `test.md` file is ignored and it should be missing from the remote repo and you can solve this issue by following the steps below:

1. First you need to clear the git tracking changes for this file by running the command `git rm --cashed path/your_file_name.extension`
2. Then add the changes by running `git add .`, then commit the changes.
3. don't forget to push the changes 👍🏼 

Hope this was helpful 💪🏼 good luck.

File: .github/workflows/linters.yml [hunk 3]
```diff
@@ -0,0 +1,56 @@
+name: Linters
```
Comment (adel-gu) [inline .github/workflows/linters.yml:1]: ### These are suggestions for future projects:

1. In future projects please ensure you have a meaningful branch name that conveys an action or a feature name that you trying to implement, for example: `setup-hello-microvers` or `hello-microverse` ...etc
2. Also ensure that the PR has a descriptive Title and summary that explains your achievements or the changes you made for the milestone, as an example:
    - **Title:** In this PR I made the following changes:
    - **Summery:**
        - Create the `.html` file.
        - Setup linters for html-css project.
        - ...etc
3.  Also ensure your repo includes meaningful commit messages, a good habit for a professional developer is committing often for each change is made 👍🏼

**Review by adel-gu — CHANGES_REQUESTED** [review]
# Status: REQUIRE CHANGES ♻️ 
Hi @cheronojudith 👋🏼 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.



 ## Highlights
- Both `.html` and `.css` files exist ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by huseyinbiyik — APPROVED** [review]
Hi @cheronojudith 

## ✔️ Status: APPROVED 🏆 🎉 🎊

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text



## Highlights
- Linter set up well and checks pass 🟢
- PR description is descriptive 📝
- Great implementation of the requested changes of previous reviewers 🕸️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bimbolabuari — COMMENTED** [review]
Hello @cheronojudith  👋,

_This is an additional review as a result of the second opinion request._

- Welcome to Microverse 🎉 
   - I can see the issue has been resolved and your project approved. 👍 
   
Keep rocking 🚀!

**Pull request merged**